tokens.xml

```diff
@@ -7058,7 +7058,7 @@ Equip {1}</text>
                 <cmc>0</cmc>
                 <pt>1/1</pt>
             </prop>
-            <set picURL="https://media-dominaria.cursecdn.com/attachments/113/426/635201279747658553.jpg">C13</set>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/6/6/66d80dd1-b944-4cb2-8578-b4dbcabbbc1e.jpg?1562916501">C13</set>
```

Comment:
this is a picture of Ophiomancer instead

Reply:
Whoops, had multiple JSONs open at once and forgot to double check which was which. Fixed.